src/vm/gdbjit.h

```diff
@@ -366,11 +366,27 @@ class NotifyGdb
         unsigned MemSize;
         MemBuf() : MemPtr(0), MemSize(0)
         {}
+        bool Resize(unsigned newSize)
```

Comment:
I am sorry, you are right. I have forgotten that the MemPtr is a holder and the diff has hidden that member from me.